Sources/BSWFoundation/APIClient/Router.swift

```diff
@@ -32,13 +32,13 @@ extension APIClient {
                 guard let url = urlRequest.url else {
                     throw APIClient.Error.encodingRequestFailed
                 }
-                guard let parameters = endpoint.parameters, !parameters.isEmpty, var urlComponents = URLComponents(url: url, resolvingAgainstBaseURL: false) else { break }
+                guard let parameters = endpoint.parameters as? [String: Any], !parameters.isEmpty, var urlComponents = URLComponents(url: url, resolvingAgainstBaseURL: false) else { break }
```

Comment:
What worries me is that this won't be as easy to port over. I'll have to figure out a way checking out Alamofire